cookbooks/kinematically_driven_subduction_2d/kinematically_driven_subduction_2d_case2b.prm

```diff
@@ -0,0 +1,46 @@
+# This cookbook models oceanic subduction in a 2d
+# box of 3000 x 670 km.
+# The subducting plate is driven by inflow boundary
+# conditions on the top part of the right boundary,
+# while a balancing outflow is prescribed underneath.
+# Only density and viscosity differ per composition.
+# Building on case 2a, an initial temperature field
+# based on the plate cooling model and
+# a mantle adiabatic gradient of 0.25 degrees/km
+# is prescribed. Compared to case2a, this model includes
+# a temperature dependent density.
+# High resolution is required to a.o. resolve the weak zone
+# between the subducting and overriding plate, so about
+# 100 MPI processes are required.
```

Comment:
I think it would be ok to replace all of the text above with:
`A modification of kinematically driven 2d subduction cookbook case 2a to include temperature-dependent density.`